ui_framework/doc_site/src/views/accessibility/accessibility_example.js

```diff
@@ -37,5 +39,27 @@ export default props => (
         <KeyboardAccessible />
       </GuideDemo>
     </GuideSection>
+
+    <GuideSection
+      title="ScreenReaderOnly"
+      source={[{
+        type: GuideSectionTypes.HTML,
+        code: screenReaderOnlyHtml,
+      }]}
+    >
+      <GuideText>
+        This class can be useful to add accessibiility to older designs that
+        are still in use, but it shouldn't be a permanent solution.
+        See <a href='http://webaim.org/techniques/css/invisiblecontent/'>http://webaim.org/techniques/css/invisiblecontent/</a> for more information.
```

Comment:
You can import the `GuideLink` component on line 8 and use it here, to get the UI Framework documentation styling:

```jsx
<GuideLink href="http://webaim.org/techniques/css/invisiblecontent/">
  http://webaim.org/techniques/css/invisiblecontent/
</GuideLink>
```